src/nyc_trees/nyc_trees/settings/base.py

```diff
@@ -47,7 +47,8 @@
         'USER': environ.get('NYC_TREES_DB_USER', 'nyc_trees'),
         'PASSWORD': environ.get('NYC_TREES_DB_PASSWORD', 'nyc_trees'),
         'HOST': environ.get('NYC_TREES_DB_HOST', 'localhost'),
-        'PORT': environ.get('NYC_TREES_DB_PORT', 5432)
+        'PORT': environ.get('NYC_TREES_DB_PORT', 5432),
+        'TEST_NAME': environ.get('NYC_TREES_TEST_DB_NAME', 'test_nyc_trees')
```

Comment:
I think that the process of setting both defaults has their own purpose. The one in Ansible guards against a scenario where the environmental variable isn't set. The one inside the Django settings file guards against a scenario where you deploy the application without our Ansible setup (unlikely, but the other settings are setup this way).